content/en/docs/setup/install/helm/index.md

```diff
@@ -230,27 +230,32 @@ $ kubectl get crd -oname | grep --color=never 'istio.io' | xargs kubectl delete
 
 You can generate the manifests for each component before installing Istio using the `helm template`
 sub-command.
-For example, use the following command to generate a manifest that can be installed with `kubectl` for the `istiod` component:
+For example, to generate a manifest that can be installed with `kubectl` for the `istiod` component:
 
 {{< text syntax=bash snip_id=none >}}
-$ helm template istiod istio/istiod -n istio-system --kube-version <Kubernetes version of target cluster> > istiod.yaml
+$ helm template istiod istio/istiod -n istio-system --kube-version {Kubernetes version of target cluster} > istiod.yaml
 {{< /text >}}
 
-The generated manifest can be used to inspect what exactly is installed as well as to track changes to the manifest over time. Note that any additional flags or values overrides you would normally use for installation should also be supplied to the `helm template` command.
+The generated manifest can be used to inspect what exactly is installed as well as to track changes to the manifest over time.
 
-The following can be used to install the manifest generated above, which will create the `istiod` component in the target cluster:
+{{< tip >}}
+Any additional flags or custom values overrides you would normally use for installation should also be supplied to the `helm template` command.
+{{< /tip >}}
+
+To install the manifest generated above, which will create the `istiod` component in the target cluster:
 
 {{< text syntax=bash snip_id=none >}}
 $ kubectl apply -f istiod.yaml
 {{< /text >}}
 
-Note that this alternative installation method may not apply the resources with the same sequencing of dependencies as `helm install` and is not tested in Istio releases.
-
 {{< warning >}}
 If attempting to install and manage Istio using `helm template`, please note the following caveats:
 
 1. The Istio namespace (`istio-system` by default) must be created manually.
 
+1. Resources may not be installed with the same sequencing of dependencies as
+`helm install` and is not tested as part of Istio releases.
```

Comment:
Do we need to give this much color here? I'm asking because I don't know, part of me thinks it might be less confusing to just say "we don't test it" and leave it at that.

Reply:
I like the bullet, kk.

>  Is this enough to dissuade people?

Not the determined. We have a big old warning block here for both forms of `templating` for a reason - this method has more footguns, which you sign up for when you pick it.